docroot/themes/humsci/humsci_basic/src/js/shared/timeline/expand-collapse-timeline.js

```diff
@@ -0,0 +1,26 @@
+// Timelines are expanded by default
+// Find when a timeline has been set to collapsed so that we can
+// adjust the default attribute values
+const timelineCollapsed = document.querySelectorAll('.hb-timeline__collapsed');
+
+// Find timeline items are are open inside of timelineCollapsed and close them!
+timelineCollapsed.forEach(timeline => {
```

Comment:
Something to think about, this functionality exists in drupal core: https://git.drupalcode.org/project/drupal/-/blob/9.2.x/core/core.libraries.yml#L164

It's very possible to just replace this entire js and require the core library instead.

Reply:
Good morning @pookmish :wave:

I've added a pointer cursor to the summary of timeline items. That was a great call! Thank you so much for that. 

The timeline item paragraph type is using the functionality from drupal core. I wrote the JS for the vertical timeline paragraph type, so that a user can choose to display all the timeline items within it as expanded (unchecked, default state) or collapsed (checked). I wasn't sure how to do that with drupal. But I'm definitely willing to learn! :smile:

Would it be possible for us to move forward with the JS functionality as it is for now and to consider refactoring in the future?  Thank you!!

Reply:
ok thanks for the information. This is fine and certainly isn't a blocker at this time.

Reply:
Awesome! thank you!!!